openapi-specs/schema.json

```diff
@@ -4918,6 +4964,20 @@
             "required": true,
             "type": "string",
             "description": "The ID of a backup. Must be URL-safe and work as a filesystem path, only lowercase, numbers, underscore, minus characters allowed."
+          },
+          {
+            "name": "bucket",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The name of the S3 bucket"
```

Comment:
can we make those descriptions more generic ones? removing the `S3` information would be sufficient in this case